react-native-gutenberg-bridge/ios/GutenbergBridgeDelegate.swift

```diff
@@ -20,12 +20,14 @@ extension Gutenberg.MediaSource {
     public static let mediaLibrary = Gutenberg.MediaSource(id: "SITE_MEDIA_LIBRARY", label: "", types: [.image, .video])
     public static let deviceLibrary = Gutenberg.MediaSource(id: "DEVICE_MEDIA_LIBRARY", label: "", types: [.image, .video])
     public static let deviceCamera = Gutenberg.MediaSource(id: "DEVICE_CAMERA", label: "", types: [.image, .video])
+    public static let mediaEditor = Gutenberg.MediaSource(id: "MEDIA_EDITOR", label: "", types: [.image])
 
     static var registeredInternalSources: [Gutenberg.MediaSource] {
         return [
             .deviceCamera,
             .deviceLibrary,
             .mediaLibrary,
+            .mediaEditor,
```

Comment:
This is meant to be used to select an image on an empty image block. Not sure if we want the Edit image option here.

Btw, I just noticed this is not working properly, and the internal sources are still defined in JS.

Reply:
Removed in 4535a1bea1d809b19e482b60f0d0cbf9d801426a